_unittest/test_02_2D_modeler.py

```diff
@@ -14,8 +14,21 @@
 
 
 class TestClass(BasisTest):
-    def setup_class(self):
-        BasisTest.setup_class(self, project_name="test_primitives", design_name="2D_Primitives", application=Maxwell2d)
+    def setup_class(self, project_name=None, design_name=None, solution_type=None, application=None):
+        """
+        redefine not to mock empty call
+        """
+
+    def setup(self):
+        super().setup_class(project_name="test_primitives", design_name="2D_Primitives", application=Maxwell2d)
```

Comment:
@maxcapodi78  Will the call to `super()` work with IronPython. FRom what I can remember in the past it used to break some ironpython tests. If yes, @beliaev-maksim you can use directly the base class name directly.

Reply:
@MaxJPRey 
change is outdated :)

Reply:
Just saw that. Thanks @beliaev-maksim

Reply:
just FYI, you should not call `BaseClass.method`

in IPY you need to run `super(MyClass, self).method()`